clearwater_config_access/bgcf_json_config_plugin.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) Metaswitch Networks 2017
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+
+"""This contains the BGCF json subclass of ConfigType"""
+
+from metaswitch.clearwater.config_manager.config_type_class_plugin import ConfigType
+
+
+class BgcfJson(ConfigType):
+    name = 'bgcf_json'
+    filetype = 'json'
+    # file_download_name is used to agree with the current naming system when
+    # writing to file.
+    file_download_name = 'bgcf.json'
+
+    # This help_info appears as user-visible help text in the usage statement
+    # for cw-config.
+    help_info = ('''bgcf_json - this stores the configuration of each
+                    BGCF.''')
+    schema = '/usr/share/clearwater/clearwater-config-manager/scripts/config_validation/bgcf_schema.json'
```

Comment:
Is this going to exists in PC? I'm not sure who creates the schemas, and whether this is not the Project Clearwater BGCF one

Reply:
I believe it's present on sprout nodes (it's in the sprout repository under the sprout-bgcf.root directory). I'll live test to make sure though.